bigtable_rs/src/bigtable.rs

```diff
@@ -403,6 +403,14 @@ impl BigTableConnection {
             timeout: self.timeout.clone(),
         }
     }
+
+    /// Provide a convenient method to update the inner `BigtableClient` so a newly configured client can be set
+    pub fn configure_inner_client(
+        &mut self,
+        config_fn: fn(BigtableClient<AuthSvc>) -> BigtableClient<AuthSvc>,
```

Comment:
Oh, one more thing, is it possible to pass in a &mut here, so we do not need to do the clone below?